### ISSUE TITLE
Please support

### ISSUE THREAD

**Issue by elnursamedzade** [marytts/marytts#1011]
Hello there. Project founders know that I have been working with marytts for months and have not been successful. I still haven't given up.
I am from Azerbaijan. My tongue is very similar to Turkish language. I am visually impaired. Will one of you do what I can't do for me?
I have audio and text files in my language. Can you train this data for me? Or can you tell me in detail how to do it by contacting the private?
Please help me. I can't get support from anyone.
I would be very happy if you help me with this. I can even make a certain amount of donations.
Please don't go through, I need your deck a lot.
Yours sincerely, Elnur from Azerbaijan.

**Comment by seblemaguer** [comment]
Hello. I transferred the issue because the plugin issue tracker is not the best place for your general question.

At which stage are you blocked? The language building part?

**Comment by elnursamedzade** [comment]
Hello @seblemaguer. I apologize for not sending the subject to the right place.
In fact, I have cloned the voice-dfki-ot plugin because the language is almost the same as the Turkish language and the build is successful.
But when I replace the wav files with my own voices and build them, I get an md5 checksum error. Is there a way I can skip this? Or any suggestions?

**Comment by seblemaguer** [comment]
How do you achieve the replacement? Have you modified the build.gradle?

**Comment by elnursamedzade** [comment]
The my build.gradle file:
`plugins {
    id 'de.dfki.mary.voicebuilding-legacy' version '5.4'
}

marytts {
    voice {
        name = 'dfki-ot'
        language = 'tr'
        gender = 'male'
        type = 'unit selection'
        description = 'A male Turkish unit selection voice'
        license {
            name = 'Creative Commons Attribution-NoDerivs 3.0 Unported'
            shortName = 'CC-BY-ND'
            url = 'http://mary.dfki.de/download/by-nd-3.0.html'
        }
        samplingRate = 16000
    }
}

wav.srcDir = file('wav')
`

**Comment by elnursamedzade** [comment]
Build is successful when I put the original wav files. But when I put my own wav files, it gives md5 checkum error.

**Comment by seblemaguer** [comment]
Could you paste the error?

**Comment by elnursamedzade** [comment]
`elka@DESKTOP-LCA50BE:~$ cd voice-dfki-ot
elka@DESKTOP-LCA50BE:~/voice-dfki-ot$ ./gradlew build
Starting a Gradle Daemon, 1 incompatible and 1 stopped Daemons could not be reused, use --status for details
> Task :wav FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Failed to capture fingerprint of input files for task ':wav' property 'srcDir' during up-to-date check.
> Failed to create MD5 hash for file '/home/elka/voice-dfki-ot/wav/ot0001.wav' as it does not exist.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with
--scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.10.3/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 11s
3 actionable tasks: 1 executed, 2 up-to-date
elka@DESKTOP-LCA50BE:~/voice-dfki-ot$`

**Comment by psibre** [comment]
> `> Failed to create MD5 hash for file '/home/elka/voice-dfki-ot/wav/ot0001.wav' as it does not exist.`

The problem is not so much "Failed to create MD5", but "it does not exist".
Where are the files you want to build your voice with?

**Comment by elnursamedzade** [comment]
my lab files: voice-dfki-ot/build/lab
my text files: voice-dfki-ot/build/text
my wav files: voice-dfki-ot/wav

**Comment by elnursamedzade** [comment]
> Could you paste the full log ?

Starting a Gradle Daemon, 1 incompatible and 1 stopped Daemons could not be reused, use --status for details
> Task :generateSource UP-TO-DATE
> Task :compileJava UP-TO-DATE
> Task :compileGroovy NO-SOURCE
> Task :processDataResources NO-SOURCE
> Task :wav FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Failed to capture fingerprint of input files for task ':wav' property 'srcDir' during up-to-date check.
> Failed to create MD5 hash for file '/home/elka/voice-dfki-ot/wav/ot0001.wav' as it does not exist.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.10.3/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 13s
3 actionable tasks: 1 executed, 2 up-to-date

**Comment by elnursamedzade** [comment]
Please come back to me. I need your help a lot.

**Comment by seblemaguer** [comment]
The error message says that there is no file `/home/elka/voice-dfki-ot/wav/ot0001.wav`. Could you paste the build.gradle file?

**Comment by elnursamedzade** [comment]
No, the file is there. Here is my

**Comment by elnursamedzade** [comment]
No, the file is there. Here is my project:
https://drive.google.com/file/d/1durO-Faivo8ccpo50KIP8q_g0uQbHN3Z/view?usp=sharing

**Comment by elnursamedzade** [comment]
Please can you build this project?

**Comment by elnursamedzade** [comment]
My build.gradle file:
`plugins {
    id 'de.dfki.mary.voicebuilding-legacy' version '5.4'
}

marytts {
    voice {
        name = 'dfki-ot'
        language = 'tr'
        gender = 'male'
        type = 'unit selection'
        description = 'A male Turkish unit selection voice'
        license {
            name = 'Creative Commons Attribution-NoDerivs 3.0 Unported'
            shortName = 'CC-BY-ND'
            url = 'http://mary.dfki.de/download/by-nd-3.0.html'
        }
        samplingRate = 16000
    }
}

wav.srcDir = file('wav')
`

**Comment by seblemaguer** [comment]
I built it and it works on my side. but I am on linux. You are on windows right?

**Comment by elnursamedzade** [comment]
Yes, windows. I am using windows 10 ubuntu terminal. Have you made any edits to the file?

**Comment by elnursamedzade** [comment]
Yes! Yes! Finally I managed to build!
I installed a virtual ubuntu operating system and everything is working fine right now.
Thank you very much! I love you!
Now we can turn it off. I will have a new question and I will open a title under a new issue.

**Comment by seblemaguer** [comment]
I still keep this opened because it may be an issue on windows and we have to check why

**Comment by elnursamedzade** [comment]
Yes, Mr. @seblemaguer currently continues with the problem I mentioned above with ubuntu terminal under windows 10. But there is no problem with the ubuntu virtual operating system. Thank you. Please can you deal with the question?

**Comment by elnursamedzade** [comment]
Hello all.
I want to share new developments with you with Windows 10.

Under Windows 10, I removed the java with the ubuntu terminal and reinstalled it with the sudo apt-get install default-jdk command.
My java version:
elka@DESKTOP-LCA50BE:~$ java -version
openjdk version "11.0.7" 2020-04-14
OpenJDK Runtime Environment (build 11.0.7+10-post-Ubuntu-3ubuntu1)
OpenJDK 64-Bit Server VM (build 11.0.7+10-post-Ubuntu-3ubuntu1, mixed mode, sharing)
elka@DESKTOP-LCA50BE:~$

This time when I enter the ./gradlew build command it gives the following error:

6 tests completed, 2 failed

> Task :integrationTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':integrationTest'.
> There were failing tests. See the report at: file:///home/elka/voice-dfki-ot/build/reports/integrationTest/index.html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with
--scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.10.3/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 52s
61 actionable tasks: 60 executed, 1 up-to-date
elka@DESKTOP-LCA50BE:~/voice-dfki-ot$

But my voice comes under build/distributions.
Why might the above error be caused by?
PS I ran the ./gradlew build command with sudo ./gradlew build and this time it doesn't give md5 error.

As I said, I am not getting an error with the virtual ubuntu OS.
